adage/analyze/management/commands/add_ml_model.py

```diff
@@ -6,7 +6,7 @@
 invoked like this:
 
   python manage.py add_ml_model <ml_model_name> <organism_tax_id> \
- [--directed_edge]
+ [--directed_edge] --g2g_edge_cutoff <cutoff_value>
```

Comment:
Actually the cutoff value option is also optional. So I will enclose this option into `[ ... ]` too.